.github/workflows/CI_cmake_build.yml

```diff
@@ -0,0 +1,51 @@
+name: C/C++ CI based on CMake
+
+on:
+  push:
+  pull_request:
```

Comment:
This looks like its the same file as on master, just named `CI_cmake_build.yml` instead of `CI_make_build.yml`

Reply:
ah ok now I understand from where the merge conflict arose...probably @jimbraun copy-pasted the workflows here in master

I think it would have been sufficient to activate them from the repo settings

Reply:
and this probablt also explains why the cmake tests are all doubled

I can remove the one in master with a subsequent PR